File: web_app/PythonProxy/Dto/FileFromCacheDto.py
from pydantic import BaseModel
from datetime import datetime

class UsersEmailsFileNames(BaseModel):
    userEmail:str
    fileName:str
    uploadTime:str

class FileFromCacheDto(BaseModel):
    base64EncFile:str
    base64Tag:str
    key:str
    iv:str
    emailsFilenames:list[UsersEmailsFileNames]

File: web_app/FileStorageApp/FileStorageApp/Shared/Dto/FileFromCacheDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FileStorageApp.Shared.Dto
{
    public class FileFromCacheDto
    {
        public string base64EncFile { get; set; }
        public string  base64Tag { get; set; }
        //public string key { get; set; }
        //public string iv { get; set; }
        //public List<UsersEmailsFilenames> emailsFilenames { get; set; }
        public List<PersonalisedInfoDto> personalisedList { get; set; }
    }
}
